testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -314,6 +314,75 @@ if(devices.real().deviceType === 'desktop') {
     });
 }
 
+QUnit.module('Accessibility', ()=> {
+
+    QUnit.test('list should have a selectByClick=true by default', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2], selectByClick: true }).dxList('instance');
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.ok(isSelectByClick);
+    });
+
+    QUnit.test('list should have to change the value of selectByClick during initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2], selectByClick: false }).dxList('instance');
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.notOk(isSelectByClick);
+    });
+
+    QUnit.test('list should have to change the value of selectByClick after updating', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2] }).dxList('instance');
+
+        list.option('selectByClick', false);
+
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.notOk(isSelectByClick);
+    });
+
+    QUnit.test('list item should not respond to a click', function(assert) {
+        let actionCount = 0;
+        const $list = $('#list').dxList({
+            items: [1, 2],
+            showSelectionControls: true,
+            selectByClick: false,
+            selectionMode: 'multiple',
+            onSelectionChanged() {
+                actionCount++;
+            },
+        });
+
+        const $item1 = $list.find(`.${LIST_ITEM_CLASS}`).eq(0);
+        const $item2 = $list.find(`.${LIST_ITEM_CLASS}`).eq(0);
```

Comment:
const $item2 = $list.find(`.${LIST_ITEM_CLASS}`).eq(1)?

Reply:
typo, thank you